program/src/execute_sale/mod.rs

```diff
@@ -294,34 +285,27 @@ pub fn handler(
         escrow_payment_bump,
         free_trade_state_bump,
         program_as_signer_bump,
-        price,
```

Comment:
Dropping the price as instruction data. Instead rely on the price on the listing.